mms/go/cmd/gocqlxapp/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"goapp/internal/log"
```

Comment:
why not std log?

Reply:
This just happened automatically and the other apps also use it.